packages/i18n/package.json

```
@@ -42,17 +42,17 @@
   },
   "dependencies": {
     "@enact/core": "^4.1.2",
-    "prop-types": "^15.7.2",
-    "ramda": "^0.24.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "xhr": "^2.4.1"
+    "prop-types": "^15.8.0",
+    "ramda": "^0.27.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "xhr": "^2.6.0"
   },
   "peerDependencies": {
-    "ilib": "^14.4.0 || ^14.6.0-webos"
+    "ilib": "^14.11.1 || ^14.11.1-webos"
```

Shouldn't it be below?

```suggestion
    "ilib": "^14.11.1 || ^14.11.1-webos1"
```